sql_runner/snowflake_target.go

```diff
@@ -1,21 +1,22 @@
 package main
 
 import (
+	"context"
 	"database/sql"
-	sf "github.com/snowflakedb/gosnowflake"
-	"log"
-	"strings"
-	"time"
-	//"github.com/olekukonko/tablewriter"
 	"errors"
 	"fmt"
 	"github.com/olekukonko/tablewriter"
+	sf "github.com/snowflakedb/gosnowflake"
+	"log"
 	"os"
+	"strings"
+	"time"
 )
 
 // Specific for Snowflake db
 const (
-	loginTimeout = 5 * time.Second // by default is 60
+	loginTimeout  = 5 * time.Second                // by default is 60
+	multiStmtName = "multiple statement execution" // https://github.com/snowflakedb/gosnowflake/blob/e909f00ff624a7e60d4f91718f6adc92cbd0d80f/connection.go#L57-L61
```

Comment:
Not following the relevance of this comment?

Reply:
Just shows where this value comes from. This is what the driver also uses to determine whether it is a multi-statement query.

Reply:
Ah ok I think I follow now. So that leads me to the follow-up - in that code it says:

> // isMultiStmt returns true if the statement type code is of type multistatement

... so can we not just call `isMultiStmt` instead of matching strings?

Reply:
It would be nicer indeed, but it is unexported name.

Reply:
Ah ok ok I follow. Grand so.